### ISSUE TITLE
[TwigBundle] FilesystemLoader::exists() throws a deprecated warning since Twig 1.27 and Symfony 3.0.5

### ISSUE THREAD

**Issue by francoispluchino** [symfony/symfony#20346]
The `Symfony\Bundle\TwigBundle\Loader\FilesystemLoader::exists()` method throws a deprecated exception:

```
In Symfony\Bundle\TwigBundle\Loader\FilesystemLoader::findTemplate(), you must accept a second argument that when set to "false" returns "false" instead of throwing an exception. Not supporting this argument is deprecated since version 1.27
```

In `Symfony\Bundle\TwigBundle\Loader\FilesystemLoader::findTemplate()`, the `$throw` argument isn't passed to the `Twig_Loader_Filesystem::findTemplate()` parent method.

The result, is that the value of `func_num_args()` is equal to 1 and not the value 2.

**Comment by xabbuh** [comment]
Thanks for reporting this @francoispluchino. This is already fixed by #20321 and will be part of the next Symfony patch releases.

**Comment by francoispluchino** [comment]
Ok, thank you, I had seen that the pull request was still merged even if it was closed.
